WorldModel/WorldModel/Core/CoreEditor.aslx

```diff
@@ -15,11 +15,9 @@
 
   <template name="EditorImageFormats">*.jpg;*.jpeg;*.png;*.gif</template>
   <template name="HTMLColorNames">AliceBlue;AntiqueWhite;Aqua;Aquamarine;Azure;Beige;Bisque;Black;BlanchedAlmond;Blue;BlueViolet;Brown;BurlyWood;CadetBlue;Chartreuse;Chocolate;Coral;CornflowerBlue;Cornsilk;Crimson;Cyan;DarkBlue;DarkCyan;DarkGoldenRod;DarkGray;DarkGrey;DarkGreen;DarkKhaki;DarkMagenta;DarkOliveGreen;Darkorange;DarkOrchid;DarkRed;DarkSalmon;DarkSeaGreen;DarkSlateBlue;DarkSlateGray;DarkSlateGrey;DarkTurquoise;DarkViolet;DeepPink;DeepSkyBlue;DimGray;DimGrey;DodgerBlue;FireBrick;FloralWhite;ForestGreen;Fuchsia;Gainsboro;GhostWhite;Gold;GoldenRod;Gray;Grey;Green;GreenYellow;HoneyDew;HotPink;IndianRed ;Indigo ;Ivory;Khaki;Lavender;LavenderBlush;LawnGreen;LemonChiffon;LightBlue;LightCoral;LightCyan;LightGoldenRodYellow;LightGray;LightGrey;LightGreen;LightPink;LightSalmon;LightSeaGreen;LightSkyBlue;LightSlateGray;LightSlateGrey;LightSteelBlue;LightYellow;Lime;LimeGreen;Linen;Magenta;Maroon;MediumAquaMarine;MediumBlue;MediumOrchid;MediumPurple;MediumSeaGreen;MediumSlateBlue;MediumSpringGreen;MediumTurquoise;MediumVioletRed;MidnightBlue;MintCream;MistyRose;Moccasin;NavajoWhite;Navy;OldLace;Olive;OliveDrab;Orange;OrangeRed;Orchid;PaleGoldenRod;PaleGreen;PaleTurquoise;PaleVioletRed;PapayaWhip;PeachPuff;Peru;Pink;Plum;PowderBlue;Purple;Red;RosyBrown;RoyalBlue;SaddleBrown;Salmon;SandyBrown;SeaGreen;SeaShell;Sienna;Silver;SkyBlue;SlateBlue;SlateGray;SlateGrey;Snow;SpringGreen;SteelBlue;Tan;Teal;Thistle;Tomato;Turquoise;Violet;Wheat;White;WhiteSmoke;Yellow;YellowGreen</template>
-
-  <type name="editor_room">
-    <isroom />
-  </type>
-  <type name="editor_object"/>
+  
+  <type name="editor_room"/>

```

Comment:
The core of the issue we're trying to fix is that these editor types get stripped out when creating a `.quest` file right?

It seems to be that the better simpler fix would be to have "actual" types to handle room vs non-room?

Reply:
May you plz explain "actual types"?

Reply:
I mean non-editor types.

Reply:
> May you plz explain "actual types"?

I think he means we should create a `defaultroom` type with `is_room` set to `true`. It would only be a default object, and `is_room` would be its only additional attribute.

So, in this bit of code we should also add a new `defaultroom` type, and [the other bit of code in DropDownTypesControl](https://github.com/textadventures/quest/pull/1316#discussion_r1899175100) would add the `defaultroom` type rather than setting `is_room`?

Reply:
There shouldn't need to be any changes to DropDownTypesControl if this is done right, I think - it should be possible to handle this entirely with inherited types.

Reply:
How would this work? an object is from type defaultobject, a room could be from a new type defaultroom and a new type defaultobjectroom for "is room and object"?

Reply:
OK I refamiliarised myself with the code.

`editor_room` and `editor_object` don't get included in `.quest` files because they're defined in `CoreEditor.aslx`, which has `type="editor"` which causes it not to be included in packaged `.quest` files.

So I think the absolute simplest fix here is to move those types into `Core.aslx`. That way there's no need to change or update anything else.

Reply:
Gotcha. Closing this and creating a new PR without all these commits in the history.

Reply:
@KVonGit Do you open a new issue for this plz?
And keep the changed code! Could there be problems with existing games? Which type is for "is object and room"?